package.json

```diff
@@ -37,7 +37,8 @@
     "@types/jest": "23.3.1",
     "@types/node": "10.9.3",
     "@types/read-pkg-up": "3.0.1",
-    "eslint": "5.4.0",
+    "eslint": "^5.4.0",
+    "eslint-plugin-react": "^7.11.1",
```

Comment:
@szeck87 pointed out to me on Slack `eslint-plugin-react` should also be added as a `peerDependency`: https://github.com/Unibeautify/beautifier-eslint/blob/221ef55a34e67079dcf35dab6a411b98360d5b6b/package.json#L33-L40